src/avfoundation.cs

```diff
@@ -9039,6 +9039,28 @@ interface AVCaptureMetadataOutputObjectsDelegate {
 	}
 #endif
 
+	[NoTV, NoMac, NoWatch, iOS (12,0)]
+	[Internal]
+	[Static]
+	interface AVCapturePhotoSettingsThumbnailFormatKeys {
+		[Field ("AVVideoCodecKey")]
+		NSString CodecKey { get; }
+
+		[Field ("AVVideoWidthKey")]
+		NSString WidthKey { get; }
+
+		[Field ("AVVideoHeightKey")]
+		NSString HeightKey { get; }
+	}
+
+	[NoTV, NoMac, NoWatch, iOS (12,0)]
+	[StrongDictionary ("AVCapturePhotoSettingsThumbnailFormatKeys")]
+	interface AVCapturePhotoSettingsThumbnailFormat {
+		NSString Codec { get; set; }
+		NSNumber Width { get; set; }
+		NSNumber Height { get; set; }
```

Comment:
What's the _real_ underlying type ? [u]integers ?
IIRC we added support to expose those as value types, c.c. @dalexsoto

Reply:
Yes, you can use the following types https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#strong-dictionary-types

Reply:
I'll double check the expected value range. Although the docs were not very clear.

Reply:
There is nothing stated in the documentation regarding the value number to add in those fields. I prefer not to take the risk in this case.